src/translations/fr.json

```diff
@@ -618,6 +618,18 @@
     "VSC": "Vsc",
     "SHUNT_COMPENSATOR": "Moyen de compensation",
     "STATIC_VAR_COMPENSATOR": "CSPR",
+    "TABULAR": "Tabulaire",
+    "TabularModification": "Modification tabulaire",
+    "ImportModifications": "Importer des modifications",
+    "TabularModificationError": "Erreur lors de la modification tabulaire",
+    "equipmentId": "ID",
+    "activePower": "p (MW)",
+    "maxActivePower": "P max (MW)",
+    "activePowerSetpoint": "P consigne (MW)",
```

Comment:
You can remove the redondent translations and just use the new keys nope ?

Reply:
Done